moto/rds2/models.py

```diff
@@ -1463,20 +1463,26 @@ def to_xml(self):
         return template.render(option_group_option_setting=self)
 
 
+def make_rds_arn(region, name):
+    return "arn:aws:rds:{0}:REDACTED:pg:{1}".format(region, name)
```

Comment:
I think you should use `moto.core.ACCOUNT_ID` here.  It seems the original reporter of the localstack issue was just trying to avoid exposing his own account id and not actually suggesting that the Arn should be redacted.